mozilla/fxa
# chore(build): remove grunt-githash

Fixes #729.

If the premise of https://github.com/mozilla/fxa/issues/729#issuecomment-480246519 is correct, it means we can remove `githash` without breaking anything. That will allow `npm run build-production` to succeed inside Docker.

To test this, I did a couple of things:

* Diffed the `dist` directories with and without this change, having run `npm run build-production` previously in each case.

* Run the content server with `npm run start-production` and check things look okay.

Is there anything else I should be checking here?

@mozilla/fxa-devs r?
